src/tests/test_dte_data_models.py

```diff
@@ -1253,151 +1376,168 @@ def test_validate_non_empty_bytes_signature_x509_cert_der(self) -> None:
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_context=False,
+            include_input=False,
+            include_url=False,
+        )
         self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        self.assertEqual(validation_errors, expected_validation_errors)
 
     def test_validate_no_leading_or_trailing_whitespace_characters_receptor_email(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('receptor_email',),
-                'msg': (
-                    "("
-                    "'Value has leading or trailing whitespace characters.', "
-                    "' fake_receptor_email@test.cl '"
-                    ")"
-                ),
+                'msg': "Value error, ('Value has leading or trailing whitespace characters.', "
+                "' fake_receptor_email@test.cl ')",
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_xml_data_1,
                 receptor_email=' fake_receptor_email@test.cl ',
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_context=False,
+            include_input=False,
+            include_url=False,
+        )
         self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        self.assertEqual(validation_errors, expected_validation_errors)
 
     def test_validate_non_empty_stripped_str_emisor_giro(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('emisor_giro',),
-                'msg': "String value length (stripped) is 0.",
+                'msg': 'Value error, String value length (stripped) is 0.',
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_xml_data_1,
                 emisor_giro='',
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_url=False,
+            include_context=False,
+            include_input=False,
```

Comment:
Sort lexicographically.

```suggestion
            include_context=False,
            include_input=False,
            include_url=False,
```